.github/workflows/python-app.yml

```
@@ -10,9 +10,8 @@ on:
     branches: [main]
 
 jobs:
-  build:
+  Build AaC Package and Docs:
```

Looks like GitHub doesn't like spaces in the name:

```suggestion
  Build-AaC-Package-and-Docs:
```